src/modules/ModuleStaking/composable.pools.ts

```diff
@@ -12,12 +12,14 @@ export function useMappedPools(props: {
   pools: Ref<undefined | null | PoolsQueryResult>
   rewards: Ref<undefined | null | Rewards<Address>>
   tokens: Ref<undefined | null | Tokens>
+  blockNumber: Ref<number | null>
```

Comment:
I've excluded `blockNumber` here because it causes re-calculations for the **whole list** each second, which is not efficient and actually not necessary.

Could you refactor it without including `blockNumber` into this computation?

Reply:
It have to be here to be able to sort pools with their active statuses. I made additional roundedBlockNumber to avoid unnecessary recalculations in main computed function

Reply:
I see.